### PR TITLE
Xenlib update domain mgmt cmds logs

### PR DESCRIPTION
**Pull request by GrygiriiS**
The intention of this PR is to update update domain mgmt cmds logs, so `xu create/pause/unpause/destroy` always produce some log output (on success and on failure), so it can be used by automatic tests like pytest.

On success log output looks like

`domain:%u created|destroyed|paused|unpaused`

### CODE REVIEW

**Comment by GrygiriiS** [comment]
Not sure why build is failed :(

File: xen-dom-mgmt/src/xen-dom-mgmt.c [hunk 1]
```diff
@@ -781,6 +781,7 @@ int domain_create(struct xen_domain_cfg *domcfg, uint32_t domid)
 			LOG_ERR("Failed to unpause domain#%u (rc=%d)", domid, rc);
 			goto stop_domain_console;
 		}
+		LOG_INF("domain:%u unpaused", domid);
```
Comment (lorc) [inline xen-dom-mgmt/src/xen-dom-mgmt.c:784]: IMO, it should have lower verbosity level. Imagine AOS use case where then can start/stop potentially tens or even hundreds of domain.

Comment (GrygiriiS) [reply xen-dom-mgmt/src/xen-dom-mgmt.c:784]: It's xenlib command line interface which will not be used by AOS. It's for demo/test/debug purposes.

Comment (lorc) [reply xen-dom-mgmt/src/xen-dom-mgmt.c:784]: I don't think that `domain_create` is part of the command line interface. This is a generic function that creates a domain.

Comment (GrygiriiS) [reply xen-dom-mgmt/src/xen-dom-mgmt.c:784]: ah ok

File: xen-dom-mgmt/src/xen-dom-mgmt.c [hunk 2]
```diff
@@ -781,7 +781,6 @@ int domain_create(struct xen_domain_cfg *domcfg, uint32_t domid)
 			LOG_ERR("Failed to unpause domain#%u (rc=%d)", domid, rc);
 			goto stop_domain_console;
 		}
-		LOG_INF("domain:%u unpaused", domid);
```
[the remark this answers is not in the document]
Comment (GrygiriiS) [reply xen-dom-mgmt/src/xen-dom-mgmt.c]: done

File: xen-dom-mgmt/src/xen-dom-mgmt.c [hunk 3]
```diff
@@ -866,8 +866,8 @@ int domain_destroy(uint32_t domid)
 
 int domain_pause(uint32_t domid)
 {
-	int rc;
 	struct xen_domain *domain = NULL;
+	int rc;
```
Comment (lorc) [inline xen-dom-mgmt/src/xen-dom-mgmt.c:870]: Unnecessary change

**Review by lorc — APPROVED** [review]
With that last comment addressed:

`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

File: xen-dom-mgmt/src/xen-dom-mgmt.c (hunk 1 above)
Comment (GrygiriiS) [reply xen-dom-mgmt/src/xen-dom-mgmt.c]: done

**Comment by lorc** [comment]
@firscity, could you please merge this?
